src/migrations/phase-1.ts

```diff
@@ -150,7 +168,7 @@ export async function deployPhase1({
     );
   }
 
-  if (startStep <= 9) {
+  if (startStep <= 10) {
```

Comment:
why are we shifting these all up by 1?

Reply:
inserted 'deploy starkware priority timelock in Step 6 since it wasn't previously deployed, so i moved up the ones after.